client/components/data/moderate-comment/index.jsx

```diff
@@ -27,17 +27,17 @@ class ModerateComment extends Component {
 		this.moderate( this.props );
 	}
 
-	componentDidUpdate( nextProps ) {
+	componentDidUpdate( prevProps ) {
 		if (
-			this.props.siteId === nextProps.siteId &&
-			this.props.postId === nextProps.postId &&
-			this.props.commentId === nextProps.commentId &&
-			this.props.newStatus === nextProps.newStatus
+			this.props.siteId === prevProps.siteId &&
+			this.props.postId === prevProps.postId &&
+			this.props.commentId === prevProps.commentId &&
+			this.props.newStatus === prevProps.newStatus
 		) {
 			return;
 		}
 
-		this.moderate( nextProps );
+		this.moderate( prevProps );
```

Comment:
One little thing, this should now be `this.props` if I'm not mistaken.

Reply:
You aren't 👍 I was being a bit hasty here. Fixed in 18f194b2ef9a69b659c9c122811a4ea8efbc0401